client_end_user/shared/src/commonMain/kotlin/data/gateway/remote/UserGateway.kt

```diff
@@ -1,20 +1,19 @@
 package data.gateway.remote
 
 import data.remote.mapper.toSessionEntity
+import data.remote.mapper.toUser
 import data.remote.model.ServerResponse
 import data.remote.model.SessionDto
+import data.remote.model.UserDto
 import domain.entity.Session
+import domain.entity.User
 import domain.gateway.IUserGateway
-import domain.utils.InvalidCredentialsException
+import domain.utils.AuthorizationException
 import io.ktor.client.HttpClient
-import io.ktor.client.call.body
 import io.ktor.client.request.forms.submitForm
-import io.ktor.client.request.header
 import io.ktor.client.request.url
 import io.ktor.http.HttpMethod
 import io.ktor.http.Parameters
-import io.ktor.http.isSuccess
-import io.ktor.http.parameters
 
 class UserGateway(client: HttpClient) : BaseGateway(client), IUserGateway {
```

Comment:
as this is the remote gatway, what about to use the name "UserRemoteGateway" for ex?